include/internal/libspdm_common_lib.h

```diff
@@ -1295,4 +1295,8 @@ static inline bool libspdm_onehot0(uint32_t mask)
     return !mask || !(mask & (mask - 1));
 }
 
+#if LIBSPDM_FIPS_MODE
```

Comment:
According to https://github.com/DMTF/libspdm/issues/1573,
1) we need to expose this API even without FIPS mode.
2) we need to expose publicly, not in internal.

Reply:
It should return a `uint32_t`.